File: C Modules/Assignments/Assignment no 6/Question-1/Type_4/Assignment no 2/queston-7.c
#include<stdio.h>
int age_verify(int age)
{
    if (age<12)
    {
        return 0;
    }
    else if (age>=12 && age<=19)
        {
       return 1;
        /* code */
        }
        else if (age >= 20 && age <= 59)
            {
            return 2;
            }
            else if (age>=60)
                {
                return 3;
                }
                
            }
int main()
{
    int age;
    printf("enter the age:");
    scanf(" %d",&age);
    int result=age_verify(age);
    if (result==0)
    {
        printf("child");
    }
    else if (result==1)
        {
        printf("Teenager");

        }
        else if (result==2)
            {
            printf("Adult");
            }
            else if (result==3)
                {
                printf("Senior");
                }
    return 0; 
}

 

File: Java Module/Assignments in c to java/Assignment-02/queston_7.java
public class queston_7 {
public static void main(String[] args) {
        int age=12;
    // printf("enter the age:");
    // scanf(" %d",&age);
    
    if (age<12)
    {
        System.out.printf("child");
        /* code */
    }
    else{
        if (age>=12 && age<=19)
        {
        System.out.printf("Teenager");
        /* code */
        }
        else{
            if (age >= 20 && age <= 59)
            {
            System.out.printf("Adult");
            }
            else
            {
                if (age>=60)
                {
                System.out.printf("Senior");
                }
                
            }
            
        }
      
    }
}
    
}